backend/api/models/annotations.py

```diff
@@ -7,6 +7,54 @@
 from django.conf import settings
 
 
+class ConfidenceIndicatorSet(models.Model):
+    """
+    This table contains collections of confidence_indicator to be used for dataset annotations.
+    An annotation_set is created by a user
+    and can be used for multiple annotation campaigns.
+    """
+
+    class Meta:
+        db_table = "confidence_sets"
+
+    def __str__(self):
+        return str(self.name)
+
+    name = models.CharField(max_length=255, unique=True)
+    desc = models.TextField(null=True, blank=True)
+
+
+class ConfidenceIndicator(models.Model):
+    """
+    This table contains confidenceIndicator to be used for dataset annotations.
+    An ConfidenceIndicatorSet is created by a user
+    and can be used for multiple annotation campaigns.
+    """
+
+    class Meta:
+        db_table = "confidence_indicator"
+
+    def __str__(self):
+        return str(self.label)
+
+    label = models.CharField(max_length=255, unique=True)
+    level = models.IntegerField()
+    confidence_indicator_set = models.ForeignKey(
+        ConfidenceIndicatorSet,
+        verbose_name="Included in this set :",
+        on_delete=models.CASCADE,
+        related_name="confidence_indicators",
+    )
+    default_confidence_indicator_set = models.ForeignKey(
```

Comment:
If I remember correctly we decided that ConfidenceIndicators would belong to a specific set, if multiple sets need the same name for an indicator then we have duplicates labels which isn't an issue. This seems to be the approach you've taken with `confidence_indicator_set = models.ForeignKey` however if so then instead of  `default_confidence_indicator_set` it's be better to have something like `is_default` boolean :thinking:

Reply:
Or to get the default from ConfidenceIndicatorSet in an easier fashion add a optional foreign key in ConfidenceIndicatorSet, though I'd rather the boolean I think.